coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +187,10 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', action='append', nargs='+',
+        help='filter --show-bears by the filter given as argument.')
```

Comment:
Write now,
`coala -B --filter-by` gives default error of `Usage : all the possible options...`.
But `coala -B --filter-by unknown` gives the following error:
`'unknown' is an invalid filter. Available filters: can_detect, can_fix, language`

IMO, this is sufficient. We don't need to show which filters are available in former query.